index.html

```diff
@@ -1,138 +1,150 @@
-    <p class="advantages-description">— №3 —<br>Все достопримечательности<br>находятся очень близко</p>
-  </li>
-  <li class="advantages-item">
-    <h3 class="advantages-title"> Красивая дорога</h3>
-    <p class="advantages-description"> — №4 —<br>Ехать в Седону из Лас-Вегаса<br> совсем не скучно!</p>
-  </li>
-  <li class="advantages-item">
-    <h3 class="advantages-title"> Мало туристов</h3>
-    <p class="advantages-description"> — №5 —<br>Большинство едет в Гранд<br> Каньон и толпится там</p>
-  </li>
-</ul>
-<section class="Search">
-  <h2 class="search-tiile">Заинтересовались?</h2>
-  <p>Укажите предполагаемые даты поездки,<br>и мы покажем вам лучшие предложения гостиниц в Седоне</p>
-<a class="search-link" href="">Поиск гостиницы в Седоне</a>
-</section>
-<section class="mailing">
-  <h2 class="mailing-tiile">Подпишитесь на рассылку</h2>
-  <p> Только полезная информация и никакого спама,<br>честное бойскаутское!</p>
-  <form class="mailing-form" action="" method="post">
-  <label class="visually-hidden" for="mailing-email">Email</label>
-  <input class="field" placeholder="email@exemple.com" type="email" name="mailing-email" id="mailing-email" required>
-  <button class="button mailing-button" type="submit">
-    <span class="visually-hidden">Подписаться</span>
-  </button>
-</form>
-</section>
-</main>
-<footer class="page-footer"
-<section class="footer-social">
-  <h2 class="visually-hidden">Давайте дружить</h2>
-  <ul class="footer-social-list">
-    <li class="footer-social-item">
-      <a class="button" href="#">
-        <span class="visually-hidden">Твиттер</span>
-      </a>
-    </li>
-    <li class="footer-social-item">
-      <a class="button" href="#">
-        <span class="visually-hidden">Фейсбук</span>
-      </a>
-    </li>
-    <li class="footer-social-item">
-      <a class="button" href="#">
-        <span class="visually-hidden">Ютьюб</span>
-      </a>
-    </li>
-  </ul>
-<a class="contacts-phone" href="tel:+78128121212">+7 (812) 812-12-12</a>
-<a class="htmlacademy-link" href="https://htmlacademy.ru/">.html academy</a>
-</section>
-</footer>
+      <section class="search">
+        <h2 class="search-tiile">Заинтересовались?</h2>
+        <p>Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+        <a class="search-link" href="">Поиск гостиницы в Седоне</a>
+      </section>
+      <section class="mailing">
+        <h2 class="mailing-tiile">Подпишитесь на рассылку</h2>
+        <p> Только полезная информация и никакого спама,честное бойскаутское!</p>
+        <form class="mailing-form" action="" method="post">
+          <label class="visually-hidden" for="mailing-email">Email</label>
+          <input class="field" placeholder="email@exemple.com" type="email" name="mailing-email" id="mailing-email"
+            required>
+          <button class="button mailing-button" type="submit" value="Подписаться"></button>
```

Comment:
Почему через `value`? Это же не инпут, а тег `<button>`.
Текст просто остается внутри кнопки:
```suggestion
          <button class="button mailing-button" type="submit">Подписаться</button>
```

Reply:
А я то думаю, почему не отображается в браузере. Упустила этот момент